swift-sdk/Internal/IterableInAppHTMLViewController.swift

```diff
@@ -0,0 +1,223 @@
+//
```

Comment:
Hmm looks like this file was removed in `inbox`.. If it has been refactored into something else, these changes should go to the new files/classes instead.

Reply:
Yeah, this was a mistaken inclusion by me, it just became `IterableHtmlMessageViewController.swift`